Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'rails', '4.2.3'
 gem 'pg'
 
 gem 'devise'
+gem "devise-async"
```

Comment:
Why we need this? Devise already has ActiveJob integration https://github.com/plataformatec/devise/#activejob-integration

Reply:
It's been so long I don't remember why. I'll look into it.

Reply:
Thank you.